yarn.lock

```diff
@@ -2432,9 +2432,9 @@ jest-validate@^21.2.1:
     leven "^2.1.0"
     pretty-format "^21.2.1"
 
-jest@^21.0.0:
+jest@^21.2.1:
   version "21.2.1"
-  resolved "https://registry.yarnpkg.com/jest/-/jest-21.2.1.tgz#c964e0b47383768a1438e3ccf3c3d470327604e1"
+  resolved "https://artifacts.apple.com/api/npm/npm-amp/jest/-/jest-21.2.1.tgz#c964e0b47383768a1438e3ccf3c3d470327604e1"
```

Comment:
@snewcomer - This seems wrong, can you fix it plz?

Reply:
This is probably due to a global `.npmrc` or something...

Reply:
welp that is embarrassing.  yep npmrc change.